integration-tests/src/test/java/org/apache/fineract/integrationtests/organization/teller/CashierTransactionsHelper.java

```diff
@@ -0,0 +1,124 @@
+        this.requestSpecification = requestSpecification;
+        this.responseSpecification = responseSpecification;
+    }
+
+    public GetTellersTellerIdCashiersCashiersIdTransactionsResponse getTellersTellerIdCashiersCashiersIdTransactionsResponse(Long tellerId,
+            Long cashierId, String currencyCode, int offset, int limit, String orderBy, String sortOrder) {
+        return ok(fineract().tellers.getTransactionsForCashier(tellerId, cashierId, currencyCode, offset, limit, orderBy, sortOrder));
+    }
+
+    public GetTellersTellerIdCashiersCashiersIdSummaryAndTransactionsResponse getTellersTellerIdCashiersCashiersIdSummaryAndTransactionsResponse(
+            Long tellerId, Long cashierId, String currencyCode, int offset, int limit, String orderBy, String sortOrder) {
+        return ok(fineract().tellers.getTransactionsWithSummaryForCashier(tellerId, cashierId, currencyCode, offset, limit, orderBy,
+                sortOrder));
+    }
+
+    public static Integer createTeller(final RequestSpecification requestSpec, final ResponseSpecification responseSpec) {
+        return (Integer) createTellerWithJson(requestSpec, responseSpec, createTellerAsJSON()).get("resourceId");
+    }
+
+    public static Map<String, Object> createTellerWithJson(final RequestSpecification requestSpec, final ResponseSpecification responseSpec,
+            final String json) {
+
+        final String url = CREATE_TELLER_URL + "?" + Utils.TENANT_IDENTIFIER;
+        return Utils.performServerPost(requestSpec, responseSpec, url, json, "");
+    }
+
+    public static String createTellerAsJSON() {
+
+        final Map<String, Object> map = getMapWithStartDate();
+
+        map.put("officeId", 1);
+        map.put("name", Utils.uniqueRandomStringGenerator("Teller 1", 5));
+        map.put("description", Utils.uniqueRandomStringGenerator("Teller For Testing", 4));
+        map.put("status", 300);
+
+        LOG.info("map :  {}", map);
+        return new Gson().toJson(map);
+    }
+
+    public static Map<String, Object> getMapWithStartDate() {
+        HashMap<String, Object> map = new HashMap<>();
+
+        map.put("locale", "en");
+        map.put("dateFormat", "dd MMMM yyyy");
+        map.put("startDate", "20 September 2011");
+
+        return map;
+    }
+
+    public static Integer createCashier(final RequestSpecification requestSpec, final ResponseSpecification responseSpec) {
+        return (Integer) createCashierWithJson(requestSpec, responseSpec, createCashierAsJSON()).get("resourceId");
+    }
+
+    public static Map<String, Object> createCashierWithJson(final RequestSpecification requestSpec,
+            final ResponseSpecification responseSpec, final String json) {
+        final String url = CREATE_CASHIER_URL + "?" + Utils.TENANT_IDENTIFIER;
+        return Utils.performServerPost(requestSpec, responseSpec, url, json, "");
```

Comment:
Don't we have generated APIs for the create APIs?

Reply:
Not that I can find. The create functions I am seeing in this class are from other helper classes but Teller and Cashier do not have helper classes. That is why these functions were created in this class.